msdk-io/msdk-io-netcdf/src/main/java/io/github/msdk/io/netcdf/NetCDFFileImportMethod.java

```diff
@@ -82,7 +85,12 @@
    * @param dataStore a {@link io.github.msdk.datamodel.datastore.DataPointStore} object.
    */
   public NetCDFFileImportMethod(@Nonnull File sourceFile) {
+    this(sourceFile, null);
+  }
+
+  public NetCDFFileImportMethod(@Nonnull File sourceFile, Predicate<MsScan> msScanPredicate) {
     this.sourceFile = sourceFile;
+    this.msScanPredicate = msScanPredicate != null ? msScanPredicate : s -> true;
```

Comment:
Can you avoid explicitly passing in a ```null``` value? What about initializing ```msScanPredicate``` to ```s -> true``` (line 68) and then using the ```Predicate.and()``` method?